plugins/available/base.plugin.bash

```diff
@@ -183,3 +183,16 @@ if ! _command_exists del; then
 		mkdir -p /tmp/.trash && mv "$@" /tmp/.trash
 	}
 fi
+
+# replace multiple file extensions at once
+function rex() {
```

Comment:
There is already a package in Ubuntu and other distros called rex and I would rather not have them clash. Could you consider naming it renex or rext?